fala/apps/cla_search/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Thanks for putting a licence in. I think it is in the wrong part of the project though. Currently you've got this in cla_search, when it should just be in the main part of the project. 

You'll need to update the README as well to link to licenece. 

[https://github.com/ministryofjustice/template-repository?tab=readme-ov-file#setup-instructions](https://github.com/ministryofjustice/template-repository?tab=readme-ov-file#setup-instructions)

Basically just copy our MOJ template in the licence area.

Reply:
I've moved the license, but not sure about linking it from the readme?